src/common/components/Button/Button.tsx

```diff
@@ -0,0 +1,32 @@
+import { ReactNode } from "react";
+import classNames from "classnames/bind";
+import styles from "./Button.module.scss";
+
+const cn = classNames.bind(styles);
+
+interface ButtonProps {
+  children: ReactNode;
+  type: "submit" | "button" | "reset";
```

Comment:
타입도 옵셔널하게 받아도 괜찮을 것 같습니다!
기본 html element의 type도 넘겨주지 않으면 default로 submit이 적용되어서요!
```suggestion
  type?: "submit" | "button" | "reset";
```